### DISCUSSION TITLE
Does Quod Libet 4.6 require libmodplug?

### DISCUSSION THREAD

**Discussion by koratfood** [quodlibet/quodlibet#4516] [Q&A] [answered]
Ubuntu 24.04's build of quodlibet fails to start if libmodplug.so.1 is not available. Installing the 'libmodplug1' package manually seems to help.

Is this caused by particularities in Ubuntu/Debian's packaging process, or is libmodplug in fact a mandatory runtime dependency for Quod Libet 4.6?

[Dump_20240414_111607.txt](https://github.com/quodlibet/quodlibet/files/14970519/Dump_20240414_111607.txt)
[quodlibet-debug.term.log](https://github.com/quodlibet/quodlibet/files/14970520/quodlibet-debug.term.log)

**Comment by koratfood** [comment]
Really? Nobody knows? Should I file this as an issue or ask elsewhere?

**Reply by declension** [reply]
It's [meant to be optional](https://github.com/quodlibet/quodlibet/blob/7729ea269ec37a6a82ca018b90a09bbc69d5719f/docs/packaging.rst#optional-runtime-dependencies), so yes this is a bug now I think

**Comment by declension** [comment, marked as answer]
Converted to issue #4528
